<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muse</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HY엽서L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>검색어를 입력해주세요.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 20px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }
QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }
QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );
    }
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-130</x>
     <y>-40</y>
     <width>551</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image/image.qrc">:/newPrefix/background.jpg</pixmap>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="buddy">
    <cstring></cstring>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources>
  <include location="image/image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
